src/settings/emaildomain/VerifyOwnershipPage.js

```diff
@@ -27,10 +27,12 @@ export type CustomDomainFailureReasonsEnum = $Values<typeof CustomDomainFailureR
 export class VerifyOwnershipPage implements WizardPageN<AddDomainData> {
 
 	oncreate(vnode: Vnode<WizardPageAttrs<AddDomainData>>) {
-		worker.getDomainValidationRecord().then(recordValue => {
-			vnode.attrs.data.expectedVerificationRecord.value = recordValue
-			m.redraw()
-		})
+		let data = vnode.attrs.data
+		data.domain.map(domainName => worker.getDomainValidationRecord(domainName).then(recordValue => {
```

Comment:
Should it ever change at this point? Why do we `map()`? If it _can_ change then we should throttle